app.test.js

```diff
@@ -0,0 +1,128 @@
+const { describe, it, expect } = require("@jest/globals");
+const request = require("supertest");
+const app = require("./app.js");
+const FS = require("fs");
+const dataBase = require("./dataBaseIndex.js");
+const { TestScheduler } = require("jest");
+let dataCheck = { full: "http://www.testurl.test" };
+
+beforeAll(async () => {
+  await dataBase.load();
+});
+
+describe("Post method tests", () => {
+  it("Should be able to get shortened URL", async () => {
+    let response = await request(app)
+      .post("/api/short/new")
+      .send({ fullUrl: "http://www.testurl.test" });
+    expect(response.status).toBe(200);
+    expect(response.text.length).toBeLessThan("http://www.testurl.test".length);
+    dataCheck.id = response.text;
+  });
+  it("Should response with an error if receiving an invalid URL", async () => {
+    let response = await request(app)
+      .post("/api/short/new")
+      .send({ fullUrl: "zcp://dfww.testcvburl.sdfsdfvvfd" });
+    expect(response.status).toBe(400);
+    expect(response.text).toBe(`{"error":"invalid url"}`);
+  });
+  it("Should response with the same id if receiving an existing URL", async () => {
+    let response = await request(app)
+      .post("/api/short/new")
+      .send({ fullUrl: `${dataCheck.full}` });
+    expect(response.status).toBe(200);
+    expect(response.text).toBe(dataCheck.id);
+  });
+});
+
+describe("Get method tests", () => {
```

Comment:
Great, but missing `400` test case for `GET` in short url route, and `400` and `404` in statistics